studies/LowerCaseIntentSchemesKillSwitch.json5

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    name: 'LowerCaseIntentSchemesKillSwitch',
+    experiment: [
+      {
+        name: 'Disabled_KillSwitch',
+        probability_weight: 100,
```

Comment:
100% RELEASE..
IMHO it's a bad practice in general, but for kill switches it could acceptable if we sure it's safe.

Reply:
yeah, I'm not sure what is the point of a kill switch in that case if it's not 100%.